.github/actions/prepare-metal-run/action.yml

```diff
@@ -0,0 +1,18 @@
+name: Prepare Metal Run
+description: "Installs Python Dependencies from cache or from PyPI if cache is not available."
+
+inputs:
+  arch:
+    description: "The architecture to use"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/download-artifact@v4
+      with:
+        name: TTMetal_build_${{ inputs.arch }}
```

Comment:
we should account for profiler build also as it has a separate tag (see build artifact yaml)

Reply:
fixed

Reply:
for reference, inputs does support a boolean type so we must use a string comparison here